lib/node_modules/@stdlib/blas/base/sdot/docs/types/index.d.ts

```diff
@@ -25,12 +25,12 @@ interface Routine {
 	/**
 	* Computes the dot product of two single-precision floating-point vectors.
 	*
-	* @param N - number of values over which to compute the dot product
+	* @param N - number of values
 	* @param x - first input array
 	* @param strideX - `x` stride length
 	* @param y - second input array
 	* @param strideY - `y` stride length
-	* @returns dot product of `x` and `y`
+	* @returns output array
```

Comment:
```suggestion
	* @returns dot product
```

@Pranavchiku The function returns the dot product, not an output array. All descriptions should be changed accordingly. This is a package where we don't need to change the returns description to something overly generic. In this case, describing the return value as the dot product is fine. We can however remove "of `x` and `y`" and shorten to just "dot product", as done in the repl.txt file.

Reply:
Ahh! My bad. Will do changes.